### PR TITLE
Enhance Survey README.md

### PR DESCRIPTION
**Pull request by permission-error**
Fixes partly https://github.com/odk-x/tool-suite-X/issues/431

This PR is still open to suggestions and modifications 

Cc @elmps2018 @wbrunette @Redeem-Grimm-Satoshi @Cveman1

### CODE REVIEW

**Review by elmps2018 — COMMENTED** [review]
@permission-error thanks for working on this! So I am not sure if it makes sense to put here all this information/links to what is in the docs, as this creates some duplication. I think the goal of the readme is about the ODK-X Survey code base, not using ODK-X survey on a tablet or designing a form. @wbrunette what do you think, does this added content belong in the ReadMe or stay in docs?

**Comment by elmps2018** [comment]
Hi @permission-error, I am having a hard time comparing to the original since you added a whole lot and then took it out, can you close this and try to push again against what was originally there to make it easier to review? Thanks!

**Comment by permission-error** [comment]
@elmps2018 I just removed most of the content since in the call we discussed most of contents should be in the docs. I just left the TOC and the FAQs

**Review by wbrunette — CHANGES_REQUESTED** [review]
As discussed on the call, the README should target people developing code. The FAQs added are quite generic and general questions. Unless the FAQ relates to "code" or "development setup" they should be in the documentation not README.

File: README.md [hunk 1]
```diff
@@ -1,12 +1,27 @@
-# Survey
+# ODK-X Survey
 
 This project is __*actively maintained*__
 
+**ODK-X Survey** is an Android application for performing data collection in the ODK-X framework.
+It operates similarly to ODK Collect, but is based on HTML, CSS, and Javascript rather than native Android, and is more flexible in its presentation and execution
```
Comment (wbrunette) [inline README.md:6]: We were trying to remove all references to ODK in the readme to help with the ODK-X/ODK confusion when people search online. Therefore, I think it's best not to do the comparison in the readme.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,28 @@
...
+  - [Running](#running)
+  - [Android Version Compatibility](#android-version-compatibility)
+- [Source Tree Information](#source-tree-information)
+- [How to Contribute](#how-to-contribute)
+- [Links for Users](#links-for-users)
+
+
+## Instructions on how to use Survey.
+
+Note: Detailed information on how to use the survey can be found [here](https://docs.odk-x.org/survey-using/).
```
Comment (elmps2018) [inline README.md:25]: better to say "how to use ODK-X survey" - it's not just one "the survey"

**Review by elmps2018 — APPROVED** [review]
This looks good to me, @wbrunette any comments?

**Review by wbrunette — APPROVED** [review]

**Pull request merged**